pyproject.toml

```diff
@@ -4,3 +4,11 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 100
+
+[tool.flake8]
```

Comment:
Probably better to have all `flake8` configurations in a single file. Since github actions and pre-commit hooks point to `setup.cfg`, maybe we put everything there?